stable/tomcat/templates/appsrv.yaml

```diff
@@ -33,16 +36,29 @@ spec:
           volumeMounts:
             - name: app-volume
               mountPath: /app
+        {{- if .Values.extraInitContainers }}
+{{ tpl .Values.extraInitContainers . | indent 8 }}
+        {{- end }}
+
       containers:
         - name: {{ .Chart.Name }}
           image: {{ .Values.image.tomcat.repository }}:{{ .Values.image.tomcat.tag }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            {{- if .Values.env }}
+{{ tpl .Values.env . | indent 12 }}
+            {{- end }}
           volumeMounts:
             - name: app-volume
               mountPath: {{ .Values.deploy.directory }}
+            {{- if .Values.extraVolumeMounts }}
+{{ tpl .Values.extraVolumeMounts . | indent 12 }}
+            {{- end }}
           ports:
             - containerPort: {{ .Values.service.internalPort }}
-              hostPort: 8009
+              {{- with .Values.hostPort }}
+              hostPort: {{ . }}
```

Comment:
I believe you're modifying the default behavior of the chart here by removing the default of `hostPort: 8009`.  Either leave the default as is, or you'll need to bump this as a breaking change.

Reply:
What about put it in values.yaml as a default value? This way, I can override it with empty value in custom values.

values.yaml:
```
hostPort: 8009
```

custom-values.yaml:
```
hostPort:
```